config-backend-next.js

```diff
@@ -1,10 +1,15 @@
 module.exports = {
-  extends: ['./config-base-next.js'],
+  extends: ['./config-base-next.js', './config-backend.js'],
   env: {
     node: true,
     mongo: true,
   },
-  rules: {
-    'node/no-unpublished-require': 'warn',
-  },
+  overrides: [
+    {
+      files: ['**/repositories/**/*', '**/models/**/*', '**/migrations/**/*', '**/scripts/**/*'],
+      rules: {
+        'unicorn/no-null': 'off',
```

Comment:
If we end up disabling this in the base config then there's no reason to disable it here